framework/src/utils/RankTwoTensor.C

```diff
@@ -28,4 +28,18 @@ mooseSetToZero<ADRankTwoTensor>(ADRankTwoTensor & v)
 {
   v.zero();
 }
+
+template <>
+RankTwoTensor
+transpose<RankTwoTensor>(const RankTwoTensor & v)
+{
+  return v.transpose();
+}
+
+template <>
+ADRankTwoTensor
+transpose<ADRankTwoTensor>(const ADRankTwoTensor & v)
+{
+  return v.transpose();
+}
```

Comment:
Yeah, I don't think those should be added. And if they were needed, why aren't they static?

Reply:
I don't recall adding these methods myself. Maybe they are the result of an careless merge.